app/jobs/teacher_training_api/create_provider_courses_job.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+module TeacherTrainingApi
+  class CreateProviderCoursesJob < ApplicationJob
+    queue_as :default
+
+    def perform(provider)
```

Comment:
We need to pass the `Provider#id` here and then retrieve the provider from the DB. This will get serialised to Redis when the job is queued.

Reply:
That's what I thought too, and I originally had it like that, but apparently it's not an issue anymore? https://github.com/DFE-Digital/register-trainee-teachers/pull/638#discussion_r592265750

Reply:
Ah right. Didn't know that. Cool 👍

Reply:
New to me too!